.github/pull_request_template.md

```diff
@@ -0,0 +1,51 @@
+<!--
+Thank you for contributing to this repository!
+
+You can help us review your changes by answering these questions.
+They are all optional, but the more information you provide the easier it will be for us to review your changes.
+Everyone here is a volunteer, so please help us out if you can.
+
+If you want to discuss anything Jupyter related, or to meet other users and developers, please say hello on https://discourse.jupyter.org/ .
+-->
+
+
+### What does this PR do?
+<!--
+Please indicate the type of change made by this PR (tick one or more of the boxes) and summarise the PR.
+Please also edit the PR title so that it contains enough context to go into a changelog.
```

Comment:
Can we give an example of what a good PR title candidate for a changelog looks/reads like?

Reply:
I'm running into the same problem I had with providing an example body:
https://github.com/jupyterhub/.github/pull/11#issuecomment-702375567

What might be nice is if the next time one of us opens a non-trivial PR we follow this template, then we could link to a couple of good examples here?

Reply:
Absolutely fine!